docs/guide.md

```diff
@@ -875,6 +875,24 @@ Notes:
   The worker grabs the job and only then verifies the throttle, this is to prevent any other worker
   from attempting to grab the job, which would have exceeded the throttle.
 
+
+#### Throttles Dependant Jobs
+
+Throttle jobs if any of its dependent jobs are running.
+
+Example:
+
+~~~ruby
+class MyJob < RocketJob::Job
+  # My job won't run if there are any dependant jobs running.
+  include RocketJob::Plugins::Job::ThrottleDependentJobs
```

Comment:
Ah, good point. Can we move the new plugin directly into the plugins directory?
I.e. Move and rename it to `RocketJob::Plugins::ThrottleDependentJobs`